tests/unit/plugins/modules/test_kubevirt_vm.py

```diff
@@ -137,8 +137,18 @@ def module_params_create():
 
 
 @pytest.fixture(scope="module")
-def k8s_module_params_create(module_params_create, vm_manifest):
+def module_params_delete(module_params_create):
```

Comment:
Why do you overlay create with delete? That should not be required?

Reply:
I've reorganized the code a bit.